README.rst

```diff
@@ -220,6 +220,7 @@ installed on your system:
 - glib
 - json-glib
 - uuid
+- iproute-dev
```

Comment:
For consistency, you may want to use  iproute instead iproute-dev, above is specified that are development versions